app/demo-stm32h7-nucleo/app-h753.toml

```diff
@@ -36,7 +36,7 @@ task-slots = ["jefe"]
 name = "drv-stm32xx-i2c-server"
 features = ["h753"]
 priority = 2
-max-sizes = {flash = 16384, ram = 2048}
+max-sizes = {flash = 16384, ram = 40964096}
```

Comment:
Seems a little excessive!

Reply:
Shockingly, this builds!

```
PROGRAM       REGION  USED   SIZE   LIMIT
kernel        flash   24000  24000  (fixed)
              ram     4212   5120   (fixed)
jefe          flash   14064  16384  16384
              ram     1856   2048   2048
sys           flash   1852   2048   2048
              ram     896    1024   1024
i2c_driver    flash   12484  16384  16384
              ram     3904   4096   40964096
packrat       flash   6440   8192   8192
              ram     1720   2048   2048
spi_driver    flash   12900  16384  16384
              ram     2064   4096   4096
net           flash   48180  65536  131072
              ram     10528  16384  16384
              sram1   12416  16384  32768
user_leds     flash   1440   2048   2048
              ram     896    1024   1024
ping          flash   4312   8192   8192
              ram     1024   1024   1024
pong          flash   596    1024   1024
              ram     896    1024   1024
udpecho       flash   8872   16384  32768
              ram     4228   8192   8192
udpbroadcast  flash   7868   8192   16384
              ram     4232   8192   8192
udprpc        flash   10892  16384  32768
              ram     4224   8192   8192
hiffy         flash   24316  32768  32768
              ram     28852  32768  32768
hf            flash   12960  16384  16384
              ram     2176   4096   4096
hash_driver   flash   8752   16384  16384
              ram     2176   4096   4096
idle          flash   84     128    128
              ram     256    256    256
rng_driver    flash   6204   8192   8192
              ram     384    512    512
dump_agent    flash   8516   16384  16384
              ram     1328   2048   2048

========== Suggested changes ==========
kernel:
  ram:    4224  (currently 5120)
```

So thanks for spotting it!